src/hint_deploy.py

```diff
@@ -363,16 +364,40 @@ def proxy_configure(container, cfg):
         docker_util.exec_safely(container, args)
 
 
+def check_hintr_online(loadbalancer, port, name):
+    result = ""
+    i = 0
+    success_message = (b'{"status":"success",'
+                       b'"errors":null,'
+                       b'"data":"Welcome to hintr"}')
+    while result != success_message and i < 10:
+        try:
+            (_, result) = docker_util.exec_safely(
+                loadbalancer,
+                ["curl", "-s", name + ":" + port]
+            )
+        except Exception as e:
+            print(e)
+            time.sleep(1)
+        i += 1
+    return result == success_message
```

Comment:
I don't think you need to check the specific message. I'd just check status code 200 as that will be more robust to future changes

Reply:
Actually I would rewrite this slightly (see also the other feedback which this also incorporates):

```
def ensure_hintr_online(loadbalancer, port, name, attempts=30):
    for i in range(attempts):
        code, output = loadbalancer.exec_run(["curl", "-s", name + ":" + port])
        if code == 0:
            return
        print(f"hintr {name} not yet ready: {output.decode('UTF-8')}")
        time.sleep(1)
    raise Exception(f"hintr worker {name} did not come up in time")
```